cmd/db.go

```diff
@@ -187,7 +187,8 @@ var (
 		Use:   "reset",
 		Short: "Resets the local database to current migrations",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return reset.Run(cmd.Context(), migrationVersion, flags.DbConfig, afero.NewOsFs())
+			skipSeed, _ := cmd.Flags().GetBool("skip-seed")
```

Comment:
Bearing in mind of future work to add config option, we can add a new option now `utils.Config.Db.Seed.Enabled = false` instead of passing down this bool through all the layers.